src/scopes/utils.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/prefer-default-export */
+import { Op } from 'sequelize';
+
+/**
+ * Takes an array of string date ranges (2020/09/01-2021/10/02, for example)
+ * and attempts to turn them into something sequelize can understand
+ *
+ * @param {String[]} dates
+ * @param {String} property
+ * @param {Op.gt || Op.lt} Operator (a sequelize date operator)
+ * @returns an array meant to be folded in an Op.and/Op.or sequelize expression
+ */
+export function compareDate(dates, property, operator) {
+  return dates.reduce((acc, date) => [
+    ...acc,
```

Comment:
Is this the idiomatic way to do this in JS? Or why did you choose this over `acc.push( blah )`?

Reply:
Pretty much on reflex, I do think it's both idiomatic and consistent generally with the rest of the app.